pkg/cluster/ipaddresses.go

```diff
@@ -16,7 +16,7 @@ import (
 	"github.com/Azure/ARO-RP/pkg/util/stringutils"
 )
 
-func (m *manager) updateRouterIP(ctx context.Context) error {
+func (m *manager) updateClusterData(ctx context.Context) error {
```

Comment:
I am almost tempted to roll this code into the PatchWithLease at the bottom of deploystorage.go.

Reply:
What is the concert here? Too many calls? 
I think this segregates details nicely if case we need to drop actions (not like we gonna do this any time soon)